services/searchService.js

```diff
@@ -9,47 +9,47 @@ const Sequelize = require('sequelize');
 const Op = Sequelize.Op;
 
 const models = require('../models');
-const utilityService = require('./utilityService');
+const { swapValues } = require('./utilityService');
 
 const env = process.env.NODE_ENV || 'development';
 const config = require('../config/config.json')[env];
 
 module.exports = {
 
-	searchSellers: (params) => {
+	searchSellers: ({ latitude, longitude, searchText }) => {
 		/**
 		 * A threshold = 0.3 gives a distance range ~ 47 KM
 		 * since latitude and longitude can be negative so check 
 		 * which one is greater after applying threshold
 		 */
 
 		return new Promise((resolve, reject) => {
-			if (!params.latitude || !params.longitude || !params.searchText) {
+			if (!latitude || !longitude || !searchText) {
 				reject('Missing params');
 			}
-
-			const latitudeMax = params.latitude + config.latitudeThreshold;
-			const latitudeMin = params.latitude - config.latitudeThreshold;
+			
+			const latitudeMax = parseFloat(latitude) + config.latitudeThreshold;
+			const latitudeMin = parseFloat(latitude) - config.latitudeThreshold;
 			let latitudeRange = [latitudeMin, latitudeMax];
 			if (latitudeMin > latitudeMax) {
 				// swap values
-				latitudeRange = utilityService.swapValues(latitudeRange);
+				latitudeRange = swapValues(latitudeRange);
 			}
 
-			const longitudeMax = params.longitude + config.longitudeThreshold;
-			const longitudeMin = params.longitude - config.longitudeThreshold;
+			const longitudeMax = longitude + config.longitudeThreshold;
+			const longitudeMin = longitude - config.longitudeThreshold;
 			let longitudeRange = [longitudeMin, longitudeMax];
 			if (longitudeMin > longitudeMax) {
 				// swap values
-				longitudeRange = utilityService.swapValues(longitudeRange);
+				longitudeRange = swapValues(longitudeRange);
 			}
 
 			const fullTextSearchQuery = 
 			        'MATCH (services.name, tags) AGAINST ("'
-			        + params.searchText
+			        + searchText
 			        + '")';
 
-			models.seller.findAll({
+			return models.seller.findAll({
```

Comment:
is return here needed?

Reply:
nah, I was testing something out, forgot to change.